packages/node_modules/@node-red/runtime/lib/storage/localfilesystem/projects/git/index.js

```diff
@@ -73,7 +75,7 @@ function runGitCommand(args,cwd,env,emit) {
             err.code = "git_missing_user";
         } else if (/nothing (add )?to commit/i.test(stdout)) {
             return stdout;
-        }
+        } 
```

Comment:
```suggestion
        }
```

No trailing spaces please